Solver_interface/include/CGAL/Eigen_diagonalize_traits.h

```diff
@@ -52,7 +52,7 @@ namespace CGAL {
 ///
 /// \cgalModels `DiagonalizeTraits`
 ///
-/// \sa http://eigen.tuxfamily.org
+/// \sa http://eigen.tuxfamily.org/index.php?title=Main_Page
```

Comment:
why is such a change needed?

Reply:
When I use in the browser:
http://eigen.tuxfamily.org
I get the page but it is automatically redirected to:
http://eigen.tuxfamily.org/index.php?title=Main_Page

Reply:
I know but why is it an issue? Does it trigger a false positive in your test?

Reply:
Indeed in the link test it gives that a redirect takes place, one can discuss about whether or not it is a false positive or not but also changes from http -> https give this type of message:

```
http://eigen.tuxfamily.org/
-> http://eigen.tuxfamily.org/index.php?title=Main_Page
  Line: 403
  Code: 301 -> 200 OK
 To do: This is a permanent redirect. The link should be updated.
```
and:
```
http://www.threadingbuildingblocks.org/
-> https://www.threadingbuildingblocks.org/
  Line: 413
  Code: 301 -> 200 OK
 To do: This is a permanent redirect. The link should be updated.
```

Reply:
100% agree for the https, my comment was only about the extra part.

Reply:
I had another look at the code change:
```
The \sc{Eigen} web site is <A HREF="http://eigen.tuxfamily.org/index.php?title=Main_Page">`http://eigen.tuxfamily.org/index.php?title=Main_Page`</A>.
```
I think I get a bit what the main objection is, the part shown to the user. I think that in this case it might be better to have:
```
The \sc{Eigen} web site is <A HREF="http://eigen.tuxfamily.org/index.php?title=Main_Page">`http://eigen.tuxfamily.org`</A>.
```
meaning that the user sees the "short" name and the reference is directly to the correct place.